src/ProjectTemplates/GenerateContent.targets

```diff
@@ -47,8 +51,8 @@
       Properties="$(GeneratedContentProperties);%(GeneratedContent.AdditionalProperties)"
       OutputPath="%(GeneratedContent.OutputPath)">
 
-      <Output TaskParameter="OutputPath" ItemName="FileWrites" />
-      <Output TaskParameter="OutputPath" ItemName="Content" />
+      <Output TaskParameter="ResolvedOutputPath" ItemName="FileWrites" />
+      <Output TaskParameter="ResolvedOutputPath" ItemName="Content" />
```

Comment:
What's this for❔

Reply:
https://github.com/dotnet/arcade/pull/7403#discussion_r645164152